package.json

```diff
@@ -13,7 +13,9 @@
     "test:watch": "jest --watch",
     "version": "json -I -f assets/manifest.json -e \"this.version='`json -f package.json version`'\" && git add assets/manifest.json",
     "package": "yarn build && web-ext build --source-dir dist --artifacts-dir out --overwrite-dest",
-    "release": "webstore upload --source dist --auto-publish",
+    "release:cws": "webstore upload --source dist --auto-publish",
```

Comment:
I'd probable use `cd dist && webstore upload --auto-publish` just to match the line at the bottom and avoid a flag.

Reply:
Good point :+1: